include/deal.II/base/point.h

```diff
@@ -195,6 +195,11 @@ class Point : public Tensor<1, dim, Number>
   DEAL_II_CUDA_HOST_DEV Number &
                         operator()(const unsigned int index);
 
+  /**
+   * Assignment operator inherited from Tensor.
+   */
+  using Tensor<1, dim, Number>::operator=;
```

Comment:
This is necessary in order to use `operator=` of Tensor for a different number. The link for [operator=](https://www.dealii.org/current/doxygen/deal.II/classTensor.html#aa4aae50e15d4cc42b73c779b484366c7)

https://en.cppreference.com/w/cpp/language/using_declaration

Reply:
I compiled the documentation with Doxygen and _as expected_ this statement does not appear in the documentation.

I wonder if it is more clear to overload `operator=` and then call the base class ` Tensor<1, dim, Number>::operator=` from `Point<dim,number>::operator=`?

Which option do you prefer: `using `or overloading `operator=`?

Reply:
I would prefer to implement
```
template <typename Number2>
Pointer& operator(Pointer<dim, Number2> &other_point);
```

The `using` declaration potentially slices the object, see https://godbolt.org/z/7W6ekT
(even if `Point` doesn't have any non-inherited member variables). Please add a changelog entry anyway.